dist/gcc-compatible/Lib_RandomBuffer_System_bindings.ml

```diff
@@ -0,0 +1,8 @@
+open Ctypes
```

Comment:
is this written by hand?

Reply:
Yes, Lib_RandomBuffer_System_bindings.ml  and _gen.ml are written by hand since the C files are also written by hand.